test/unix/test_lwt_bytes.ml

```diff
@@ -54,6 +54,11 @@ let udp_server_client_exchange server_logic client_logic =
   in
   Lwt_main.run (Lwt.join [client (); server ()])
 
+let gen_buf n =
+  let buf = Lwt_bytes.create n in
+  let () = Lwt_bytes.fill buf 0 n '0' in
```

Comment:
Technically, this is filling it with `0x30`s, for zeroes you'd want `\x00`. But it doesn't matter.